### PR TITLE
Add `udp` protocol

### PR DESCRIPTION
**Pull request by aWN4Y25pa2EK**
Enable `udp` protocol for `celestia-node`

### CODE REVIEW

File: charts/celestia-node/templates/service.yaml [hunk 1]
```diff
@@ -103,7 +116,7 @@ spec:
   selector: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/component: node
 ---
-{{- end }}
+  {{- end }}
```
Comment (smuu) [inline charts/celestia-node/templates/service.yaml:119]: nit: this should not be indented

File: charts/celestia-node/templates/statefulset.yaml [hunk 2]
```diff
@@ -188,6 +188,10 @@ spec:
           ports:
             - name: p2p
```
Comment (smuu) [inline charts/celestia-node/templates/statefulset.yaml:189]: Q: should we keep the `TCP` port name to `p2p`, or should we be explicit and name it `p2p-tcp`?

**Review by smuu — CHANGES_REQUESTED** [review]

File: charts/celestia-node/templates/statefulset.yaml (hunk 2 above)
Comment (aWN4Y25pa2EK) [reply charts/celestia-node/templates/statefulset.yaml]: yeah makes sense to be consistent and have them clearly separated. I've appended `tcp` to the initial ones

File: charts/celestia-node/templates/service.yaml (hunk 1 above)
Comment (aWN4Y25pa2EK) [reply charts/celestia-node/templates/service.yaml]: fix

**Review by smuu — CHANGES_REQUESTED** [review]
Here the rename is missing: https://github.com/celestiaorg/helm-charts/pull/3/files#diff-8604d29a488bc8012ca9edd1590f40837293ad9f3dc92b0cc42857dfc2a32fabR35

**Review by aWN4Y25pa2EK — COMMENTED** [review]
changed

**Review by smuu — APPROVED** [review]

**Pull request merged**